bin/utils/convert-argv.js

```diff
@@ -6,7 +6,7 @@ const prepareOptions = require("./prepareOptions");
 const webpackConfigurationSchema = require("../config/webpackConfigurationSchema.json");
 const validateSchema = require("webpack").validateSchema;
 const WebpackOptionsValidationError = require("webpack").WebpackOptionsValidationError;
-const findup = require("findup-sync");
+const { webpackConfigPath } = require("../../packages/utils/webpack-config-path");
```

Comment:
This should be imported from `@webpack-cli/utils/webpack-config-path` but won't it need for the package to be published? Tell me when I should change it.

Reply:
@ematipico @dhruvdutt is this the right direction? 🤔

Reply:
The utils folder is a totally different project (it has its own package.json). Because of this it must be treated as a third party library. So you have to import your utility like this:

```js
const { myUtility } = require('@webpack-cli/utils');
```

So you have to make sure that utils project exports correctly the function